src/tdastro/base_models.py

```diff
@@ -44,6 +64,27 @@ def __str__(self):
         else:
             return self.__class__.__name__
```

Comment:
I would add a module name, like mentioned in the comment above

Reply:
Added